docs/cuproj/source/user_guide/cuproj_api_examples.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c5fdf490-fa77-4e56-92d1-53101fff75ba",
+   "metadata": {},
+   "source": [
+    "# cuProj Python User's Guide\n",
+    "\n",
+    "cuProj is a GPU-accelerated Python library for cartographic coordinate projection and coordinate transformations between coordinate reference systems (CRS). The cuProj Python API provides an accessible interface to high-performance projections accelerated by CUDA-enabled GPUs. The API closely follows the [PyProj](https://pyproj4.github.io/pyproj/stable/) API."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "caadf3ca-be3c-4523-877c-4c35dd25093a",
+   "metadata": {},
+   "source": [
+    "## Contents\n",
+    "\n",
+    "This guide provides a working example for all of the python API components of cuProj.  \n",
+    "The following list links to each subsection.\n",
+    "\n",
+    "* [Installing cuProj](#Installing-cuProj)\n",
+    "* [Transformations with Transformer](#Transformations-with-Transformer)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "115c8382-f83f-476f-9a26-a64a45b3a8da",
+   "metadata": {},
+   "source": [
+    "## Installing cuProj\n",
+    "Read the [RAPIDS Quickstart Guide](  https://rapids.ai/start.html     ) to learn more about installing all RAPIDS libraries, including cuProj.\n",
+    "\n",
+    "If you are working on a system with a CUDA-enabled GPU and have CUDA installed, uncomment the  \n",
+    "following cell and install cuSpatial:"
```

Comment:
```suggestion
    "If you are working on a system with a CUDA-enabled GPU and have CUDA installed, uncomment the following cell and install cuSpatial:"
```

The newline makes it a little awkward on my machine - `the` got its own line, which makes me assume word-wrapping works and we should make this a single long line and let Sphinx figure it out.

Reply:
I actually think that this whole cell is a bit weird. If they are looking at the docs online, they can't run this. If they are looking at it in the notebook, then they should only run this if they don't already have rapids installed.